package.json

```diff
@@ -7,7 +7,7 @@
     "main": "./dist",
     "files": [
         "dist/**/*",
-        "README",
+        "README.md",
         "LICENSE"
```

Comment:
Readme and license are included by default in `yarn pack` 🤔

Reply:
Interesting, I'll remove them from there then 👍

Reply:
This is in NPM convention:

![image](https://user-images.githubusercontent.com/22725671/94153023-8d67a080-fe7c-11ea-8c5f-db024e27e1fd.png)
https://npm.github.io/publishing-pkgs-docs/publishing/the-npmignore-file.html